designer/server/src/main/scala/pl/touk/nussknacker/ui/process/processingtypedata/DefaultProcessingTypeDeploymentService.scala

```diff
@@ -9,4 +10,7 @@ import scala.concurrent.{ExecutionContext, Future}
 class DefaultProcessingTypeDeploymentService(processingType: ProcessingType, deploymentService: DeploymentService) extends ProcessingTypeDeploymentService {
   override def getDeployedScenarios(implicit ec: ExecutionContext): Future[List[DeployedScenarioData]] =
     deploymentService.getDeployedScenarios(processingType)
+
+  override def markProcessFinishedIfLastActionDeploy(processName: ProcessName)(implicit ec: ExecutionContext): Future[Option[ProcessAction]] =
+    deploymentService.markProcessFinishedIfLastActionDeploy(processName)
```

Comment:
We should pass `processingType` into `deploymentService.markProcessFinishedIfLastActionDeploy` and verify inside that `processingType` for fetched ` BaseProcessDetails` matches this processing type

Reply:
Fixed (but not sure if it looks good...)